.github/workflows/docs.yml

```diff
@@ -0,0 +1,22 @@
+name: Docs
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12.x'
+      - name: Install dependencies
+        run: npm install
+      - name: Build Storybook
+        run: npm run build-storybook && mv storybook-static docs
```

Comment:
You could drop the `mv` part by using `npm run build-storybook -o ./docs`, if that makes sense.

Reply:
Oh, I tried that but it wasn't working. In fact I rather need to add it in the `package.json` like this:
```diff
diff --git a/package.json b/package.json
index a326e6d..13fad7e 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     "lint": "eslint src/iotMapManager/**/*.ts",
     "test": "echo \"Error: no test specified\" && exit 1",
     "storybook": "start-storybook -s ./src/iotMapManager/img:/assets/img -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook -o ./docs
```